frontends/article/article-page.tsx

```diff
@@ -47,14 +47,6 @@ export const Article: FC<IArticleProps> = ({
     wasLikedBefore && setLikeCount((prev) => prev - 1);
   };
 
-  useEffect(() => {
```

Comment:
![IMG_D1DB1DBF1CDC-1](https://github.com/Juratbek/udas_frontend/assets/55176773/fe6ee218-e883-4cbb-acf1-59aba4b2cb17)
some articles are not shown properly after editor is ready

Reply:
Fixed, please check it